src/org/opendatakit/briefcase/operations/CliEventsCompanion.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.operations;
+
+import java.util.function.Consumer;
+import org.apache.commons.logging.Log;
+import org.bushe.swing.event.EventBus;
+import org.opendatakit.briefcase.model.ExportProgressEvent;
+import org.opendatakit.briefcase.model.ExportSucceededEvent;
+import org.opendatakit.briefcase.model.FormStatusEvent;
+import org.opendatakit.briefcase.model.TransferSucceededEvent;
+
+/**
+ * This class will handle Events while we change from an annotation based,
+ * static API Event bus to something more flexible.
+ */
+class CliEventsCompanion {
+  static void attach(Log log) {
+    on(ExportProgressEvent.class, event -> {
+      if (event.getText().contains("Cause:")) {
```

Comment:
Looking for a string in a string here (to determine severity?) concerns me a bit, but I see from the comment above this code will be replaced.

Reply:
Yeah, also, that's part of the old event-based export progress notification system. That code should throw an exception instead of publishing an `ExportProgressEvent`... One day...